openedx/core/djangoapps/credit/models.py

```diff
@@ -29,10 +29,47 @@ class CreditProvider(TimeStampedModel):
     get credit for course. Eligibility duration will be use to set duration
     for which credit eligible message appears on dashboard.
     """
+    provider_id = models.CharField(
+        max_length=255,
+        db_index=True,
+        unique=True,
+        help_text=ugettext_lazy(
+            "Unique identifier for this credit provider. "
+            "Only alphanumeric characters and hyphens (-) are allowed. "
+            "The identifier is case-sensitive."
+        )
+    )
+
+    active = models.BooleanField(
+        default=True,
+        help_text=ugettext_lazy("Whether the credit provider is currently enabled.")
```

Comment:
I was thinking it would be confusing to alias to `_` if in the future someone tries to translate a string in an instance method (where they'd want `ugettext` instead of `ugettext_lazy`).